backend/app/views/spree/admin/variant_image_rules/index.html.erb

```diff
@@ -0,0 +1,94 @@
+<%= render :partial => 'spree/admin/shared/product_sub_menu' %>
```

Comment:
If we're going to be putting new code in I'd love for us to be using newer syntax:

```erb
<%= render "spree/admin/shared/product_sub_menu" %>
```